docs/products/kafka/howto/manage-topics-details.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Manage Apache Kafka topics in detail
+sidebar_label: Detailed topic management
+limited: true
+---
+
+Explore advanced topic management features in the Apache Kafka topic catalog. Navigate through various options and configurations for your Apache Kafka topics.
+
+## Topic overview page
+
+1. In the Apache Kafka topic catalog, click the name of a topic to open the topic details
+   pane.
+1. Click **Open topic page** to open the topic overview page.
```

Comment:
This will be disabled if you do not have 'Project Access'
The reason for this is we can show top level information to everyone but getting the deeper level config available in this view requires project access. At the moment it throws an Unauthorized exception if you open this view without the right project access.

As per https://github.com/aiven/aiven-core/pull/49376

Reply:
Added a note: 
If you are not a member of the project associated with the topic, the  **Open topic page** option is disabled.